test/integration/remote_installer_int_test.go

```diff
@@ -25,6 +30,25 @@ func (suite *RemoteInstallIntegrationTestSuite) TestInstall() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
+	// Setup running the remote installer in restricted powershell mode.
+	if runtime.GOOS == "windows" {
+		getPolicy := func() string {
+			policy, err := exec.Command("powershell.exe", "Get-ExecutionPolicy").CombinedOutput()
+			suite.Require().NoError(err, "error getting policy: "+string(policy))
+			return strings.TrimSpace(string(policy))
+		}
+		setPolicy := func(policy string) {
+			output, err := exec.Command("powershell.exe", "Set-ExecutionPolicy", "-ExecutionPolicy", policy).CombinedOutput()
+			suite.Require().NoError(err, "error setting policy: "+string(output))
+		}
+
+		policy := getPolicy()
+		defer setPolicy(policy)
```

Comment:
Could we make this a defer func and ensure the policy is set back to the default once we're done. Just to make sure it doesn't affect other tests.

Reply:
I don't understand. I am deferring resetting the policy to its default. Line 45 fetches the current default policy, and line 46 defers setting it to that default when we're done.

Reply:
What I mean is something like this:
```
defer func() {
	setPolicy(policy)
	suite.Assert().Equal(policy, getPolicy(), "should have set powershell policy to '"+policy+"'")
}()
```

Just to _ensure_ the policy has been reverted. If you feel this is unnecessary or overkill that's fine.

Reply:
Okay, thanks for clarifying. I've added your suggested assert.